<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimuladorWindow</class>
 <widget class="QMainWindow" name="SimuladorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1913</width>
    <height>1106</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <weight>75</weight>
    <italic>true</italic>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>SimuladorWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="QueueTextField">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>755</y>
      <width>381</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="QueueLabel">
    <property name="geometry">
     <rect>
      <x>1540</x>
      <y>715</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Gente en cola: </string>
    </property>
   </widget>
   <widget class="QLabel" name="QueueSizeLabel">
    <property name="geometry">
     <rect>
      <x>1680</x>
      <y>705</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonClientGenerator">
    <property name="geometry">
     <rect>
      <x>1730</x>
      <y>715</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Generar Clientes</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ClientAssignerTextFiel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>515</y>
      <width>381</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="AssignTablesButton">
    <property name="geometry">
     <rect>
      <x>1740</x>
      <y>469</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Asignar Cientes</string>
    </property>
   </widget>
   <widget class="QLabel" name="AssignerLabel">
    <property name="geometry">
     <rect>
      <x>1540</x>
      <y>480</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Recepcionista</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="TableInfoTextFiedl">
    <property name="geometry">
     <rect>
      <x>1523</x>
      <y>264</y>
      <width>381</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>8</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="TableLabel">
    <property name="geometry">
     <rect>
      <x>1540</x>
      <y>220</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Mesa #</string>
    </property>
   </widget>
   <widget class="QLabel" name="TableIDLabel">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>230</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Table16Button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>870</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Table17Button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>870</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Table18Button">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>870</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Table19Button">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>870</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Table20Button">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>870</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Table11Button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>670</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Table15Button">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>670</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Table12Button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>670</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Table13Button">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>670</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Table14Button">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>670</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Table2Button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>270</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Table6Button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>470</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Table10Button">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>470</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Table7Button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>470</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Table8Button">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>470</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Table4Button">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>270</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Table5Button">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>270</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Table3Button">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>270</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Table9Button">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>470</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Table1Button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>info</string>
    </property>
   </widget>
   <widget class="QLabel" name="chefLabel1">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>750</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="chefLabel1_2">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>180</y>
      <width>171</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="chefLabel1_3">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>470</y>
      <width>171</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ChefDessertLabel">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>715</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Chef Postres</string>
    </property>
   </widget>
   <widget class="QLabel" name="ChefMealLabel">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>440</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Chefs Platos Fuertes</string>
    </property>
   </widget>
   <widget class="QLabel" name="ChefEntranceLabel">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>150</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Chef Entradas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CookDessertButton">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>830</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cocinar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CookMealButton1">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>620</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cocinar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CookMealButton2">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>570</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cocinar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CookMealButton3">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>520</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cocinar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CookEntranceButton">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>260</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cocinar</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ChefTextField">
    <property name="geometry">
     <rect>
      <x>1523</x>
      <y>70</y>
      <width>381</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <italic>false</italic>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ChefTextLabel">
    <property name="geometry">
     <rect>
      <x>1540</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Chefs:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1913</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMeseros">
    <property name="title">
     <string>Meseros</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionMesero_1"/>
    <addaction name="actionMesero_2"/>
    <addaction name="actionMesero_3"/>
    <addaction name="actionMesero_4"/>
    <addaction name="actionMesero_5"/>
    <addaction name="actionMesero_6"/>
    <addaction name="actionMesero_7"/>
    <addaction name="actionMesero_8"/>
    <addaction name="actionMesero_9"/>
    <addaction name="actionMesero_10"/>
    <addaction name="actionMesero_11"/>
    <addaction name="actionMesero_12"/>
    <addaction name="actionMesero_13"/>
    <addaction name="actionMesero_14"/>
    <addaction name="actionMesero_15"/>
    <addaction name="actionMesero_16"/>
    <addaction name="actionMesero_17"/>
    <addaction name="actionMesero_18"/>
    <addaction name="actionMesero_19"/>
    <addaction name="actionMesero_20"/>
   </widget>
   <widget class="QMenu" name="menuTiempos">
    <property name="title">
     <string>Tiempos</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuReset">
    <property name="title">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuReservas">
    <property name="title">
     <string>Reservas</string>
    </property>
   </widget>
   <addaction name="menuMeseros"/>
   <addaction name="menuTiempos"/>
   <addaction name="menuReset"/>
   <addaction name="menuMenu"/>
   <addaction name="menuReservas"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMesero_1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Mesero 1</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMesero_2">
   <property name="text">
    <string>Mesero 2</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMesero_3">
   <property name="text">
    <string>Mesero 3</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMesero_4">
   <property name="text">
    <string>Mesero 4</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMesero_5">
   <property name="text">
    <string>Mesero 5</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMesero_6">
   <property name="text">
    <string>Mesero 6</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMesero_7">
   <property name="text">
    <string>Mesero 7 </string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMesero_8">
   <property name="text">
    <string>Mesero 8</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMesero_9">
   <property name="text">
    <string>Mesero 9</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMesero_10">
   <property name="text">
    <string>Mesero 10</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMesero_11">
   <property name="text">
    <string>Mesero 11</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMesero_12">
   <property name="text">
    <string>Mesero 12</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMesero_13">
   <property name="text">
    <string>Mesero 13</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMesero_14">
   <property name="text">
    <string>Mesero 14</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMesero_15">
   <property name="text">
    <string>Mesero 15</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMesero_16">
   <property name="text">
    <string>Mesero 16</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMesero_17">
   <property name="text">
    <string>Mesero 17</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMesero_18">
   <property name="text">
    <string>Mesero 18</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMesero_19">
   <property name="text">
    <string>Mesero 19</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMesero_20">
   <property name="text">
    <string>Mesero 20</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
